lib/carrierwave/mongoid.rb

```diff
@@ -104,9 +105,126 @@ def serializable_hash(options=nil)
         end
       RUBY
     end
+
+    def mount_uploaders(column, uploader=nil, options={}, &block)
+      field options[:mount_on] || column, type: Array, :default => []
+
+      # super
```

Comment:
The current version of `carrierwave-mongoid` depends on `"carrierwave", [">= 0.8.0", "< 0.12.0"]` https://github.com/carrierwaveuploader/carrierwave-mongoid/blob/v0.10.0/carrierwave-mongoid.gemspec#L23. The greatest version of `carrierwave` that falls within those bounds is `0.11.2` which does not have a `mount_uploaders` method defined within `CarrierWave::Mount` https://github.com/carrierwaveuploader/carrierwave/blob/v0.11.2/lib/carrierwave/mount.rb#L137. So there is no `mount_uploaders` method to call `super` on. It seems that this gem needs to be updated to work with `carrierwave 1.0.0` #172 which has the `mount_uploaders` method https://github.com/carrierwaveuploader/carrierwave/blob/v1.0.0/lib/carrierwave/mount.rb#L287